front/src/applications/operationalStudies/hooks/useScenarioData.ts

```diff
@@ -341,18 +328,18 @@ const useScenarioData = (scenario: ScenarioResponse, infraId: number) => {
     () => ({
       timetableItemsWithDetails,
       timetableItems,
-      projectionData:
-        pathUsedForProjection && projectionPath
-          ? {
-              path: pathUsedForProjection,
-              ...projectionPath,
-              projectedTrains,
-              projectionLoaderData: {
-                allTrainsProjected,
-                totalTrains: timetableItems?.length ?? 0,
-              },
-            }
-          : undefined,
+      projectionData: projectionPath?.path
```

Comment:
If you made the type change from the above comment, you should be able to do that
```suggestion
      projectionData: projectionPath
```